README.md

```diff
@@ -1,3 +1,5 @@
+[![CircleCI](https://circleci.com/gh/arenadotio/pgx.svg?style=shield)](https://circleci.com/gh/arenadotio/pgx)
```

Comment:
Can remove the comment from line 16 now that this is added.

Reply:
Done